src/motor_hardware.cc

```diff
@@ -202,6 +212,53 @@ void MotorHardware::requestVersion() {
     motor_serial_->transmitCommand(version);
 }
 
+
+// Due to greatly limited pins on the firmware processor the host figures out the hardware rev and sends it to fw
+// The hardware version is 0x0000MMmm  where MM is major rev like 4 and mm is minor rev like 9 for first units.
+// The 1st firmware version this is set for is 32, before it was always 1
+void MotorHardware::setHardwareVersion(int32_t hardware_version) {
+    ROS_ERROR("setting hardware_version to %x", (int)hardware_version);
```

Comment:
Not a ROS_ERROR, a ROS_INFO at most

Reply:
Ok but it is modeled after pre-existing deadman timer so I just did the same.  I'll convert to INFO for the new changes.